Source/Icebreaker/Bot/IcebreakerBot.cs

```diff
@@ -314,6 +332,43 @@ private async Task HandleMessageActivityAsync(ITurnContext turnContext, Cancella
             }
         }
 
+         /// <summary>
+        /// Handle profile updates from users.
+        /// </summary>
+        /// <param name="activity">Message from submitted card</param>
+        /// <param name="turnContext">Context object containing information cached for a single turn of conversation with a user.</param>
+        /// <param name="cancellationToken">Propagates notification that operations should be canceled.</param>
+        /// <returns>Tracking Task</returns>
+        private async Task OnProfileSaveAsync(Activity activity, ITurnContext turnContext, CancellationToken cancellationToken)
+        {
+            var cardPayload = JToken.Parse(activity.Value.ToString());
+            var cardAction = cardPayload["action"].Value<string>().ToLowerInvariant();
+            var userInfo = await this.dataProvider.GetUserInfoAsync(activity.From.AadObjectId);
+            switch (cardAction)
+            {
+                case "update":
+
```

Comment:
nit: remove blank line here, add blank lines before line 374 (switch) and line 366 (default case)